templates/CRM/common/TabHeader.tpl

```diff
@@ -9,7 +9,7 @@
 *}
 {* enclose all tabs and its content in a block *}
 <div class="crm-block crm-content-block">
-  {if $tabHeader and count($tabHeader)}
+  {if !empty($tabHeader) and count($tabHeader)}
```

Comment:
This is redundant. `count()` and `!empty()` both return the same truthy/falsy as far as this `{if}` is concerned.
But whatever this is fine to merge.

Reply:
yeah I might have lost the will to live before getting to the bottom of that...